core/src/worker/activities.rs

```diff
@@ -374,7 +375,21 @@ impl WorkerActivityTasks {
     }
 
     /// Called when there is a new act task about to be bubbled up out of the manager
-    fn about_to_issue_task(&self, task: PermittedTqResp) -> ActivityTask {
+    fn about_to_issue_task(&self, task: PermittedTqResp, is_eager: bool) -> ActivityTask {
+        if let Some(ref act_type) = task.resp.activity_type {
+            if let Some(ref wf_type) = task.resp.workflow_type {
```

Comment:
You should be able to combine these with the new let chains feature! https://rust-lang.github.io/rfcs/2497-if-let-chains.html 

`if let Some(ref act_type) = task.resp.activity_type && let Some(ref wf_type) = task.resp.workflow_type {`

Reply:
I didn't know that was stable, thanks!

Reply:
It's unstable:

https://github.com/rust-lang/rust/issues/53667

Reply:
Bleh, they stabilized part of it but I have no idea which part lol because every time I try to use it I fail